docker/compose/docker-compose.yml

```diff
@@ -7,10 +7,10 @@
 #
 #     http://www.apache.org/licenses/LICENSE-2.0
 #
-version: '2'
+version: '3'
 services:
   zookeeper:
-    image: zookeeper:3.5.4-beta
```

Comment:
Please do not sneak in changes like this. If you want to upgrade dependencies, do it in another PR.

Reply:
The Zookeeper operator for K8s upgraded Zookeeper to `3.5.5` in Oct 2019: https://github.com/pravega/zookeeper-operator/pull/83. I noticed this anomaly where these Compose files still refers to an older/`beta` version of Zookeeper, and so I made this change. If you guys prefer, I can raise another PR.